website/content/api-docs/system/enable-unseal-recovery.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: api
+page_title: /sys/init-unseal-recovery - HTTP API
+description: The `/sys/init-unseal-recovery` endpoint is used to establish unseal with recovery keys.
+---
+
+# `/sys/init-unseal-recovery`
+
+The `/sys/init-unseal-recovery` endpoint is used to enable the unsealing of
+Vault via recovery keys for clusters that were initialized prior to version 1.13
+or which were initialized with `disable_unseal_recovery` true.
+This feature can only be used if using an automatic unseal mechanism with recovery
+keys.  Once successful, auto-seals can be placed in 
+[unseal recovery mode](/docs/concepts/seal#unseal-recovery-mode).
```

Comment:
```suggestion
[unseal recovery mode](/vault/docs/concepts/seal#unseal-recovery-mode).
```
Hello 👋🏻 Now that the [link migration work](https://github.com/hashicorp/vault/pull/18696) has been merged, I'm visiting open PRs to resolve legacy links. All links should now use the full path from the root of developer.hashicorp.com. Let me know if you have any questions!